tests/bayesian/test_baysian_cp_full.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+import numpy as np
+import pprint as pp
+
+from htmresearch.frameworks.layers.l2_l4_inference import L4L2Experiment
+from htmresearch.frameworks.layers.object_machine_factory import (
+  createObjectMachine
+)
+
+class BayesianCPTest(unittest.TestCase):
+    def setUp(self):
+        # params
+        maxNumSegments = 2
+        L2Overrides = {
+            "learningRate": 0.1,
+            "noise": 0.0001,
+            "cellCount": 256,  # new: 256 # original: 4096
+            "inputWidth": 8192,  # new: 8192 # original: 16384 (?)
+            "sdrSize": 5,
+            "activationThreshold": 0.01,
+        }
+
+        L4Overrides = {
+            "learningRate": 0.1,
+            "noise": 0.01,
```

Comment:
The noise should be small in any case to my knowledge. More something like `1e-8` since it is introduce to prevent taking the logarithm of zero

Reply:
yes we can change that, did not have a big influence on the simple tests for column pooler

Reply:
Changed that